src/ansys/fluent/core/_version.py

```diff
@@ -6,7 +6,7 @@
 """
 
 # major, minor, patch
-version_info = 0, 12, 3
+version_info = 0, 12, 5
```

Comment:
This was not incremented during 0.12.4 release (which is in github but not in pypi)